include/votca/csg/basebead.h

```diff
@@ -113,24 +131,23 @@ class BaseBead {
   }
 
   /** does this configuration store positions? */
-  bool HasPos() const { return bead_position_set_; }
+  bool HasPos() const noexcept { return bead_position_set_; }
 
   /** set has position to true */
-  void HasPos(bool true_or_false) { bead_position_set_ = true_or_false; }
+  void HasPos(const bool &true_or_false) noexcept {
+    bead_position_set_ = true_or_false;
+  }
 
  protected:
-  BaseBead()
-      : topology_item_(nullptr),
-        molecule_item_(nullptr),
-        mass_(0.0),
-        bead_position_set_(false){};
+  BaseBead() : topology_item_(nullptr), mass_(0.0), bead_position_set_(false){};
```

Comment:
you can initialize them below in th elist

Reply:
What do you mean?

Reply:
I would set the values of these things not in the constructor but when they are declared in the private section of the class.

Reply:
change please